src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -893,6 +893,9 @@ public abstract partial class StringComparer : System.Collections.Generic.ICompa
         public static System.StringComparer InvariantCultureIgnoreCase { get { throw null; } }
         public static System.StringComparer Ordinal { get { throw null; } }
         public static System.StringComparer OrdinalIgnoreCase { get { throw null; } }
+#if netcoreapp
+        public static System.StringComparer FromComparison(System.StringComparison comparisonType) { throw null; }
+#endif
```

Comment:
I don't think we add ifdefs anymore - AFAIK we build the ref only for netcoreapp, so just remove the ifdefs and it should be fine.
Here's example: #16996

Note: I may be wrong (I didn't try it myself so far :() check the configuration of the package modified in #16996, maybe there is more to it ... cc @danmosemsft @weshaggard who know more about it

Once we clarify the steps, how-to here, please update our new contrib docs on wiki [Expose API in CoreFX](https://github.com/dotnet/corefx/wiki/Contributions#expose-api-in-corefx)